src/Tribe/Ticket_Object.php

```diff
@@ -72,6 +72,13 @@ class Tribe__Tickets__Ticket_Object {
 		public $provider_class;
 
 		/**
+		 * Holds the SKU for the ticket
+		 *
+		 * @var string
+		 */
+		public $sku;
```

Comment:
After playing with it - I believe so. We're accessing it through the `WC_Product_Simple::get_sku()` function and it's more complicated than just setting up a getter.